includes/export-action.php

```diff
@@ -149,7 +149,7 @@ function generate_email_totals(){
                         }
                         let non_empty_values = v.contact_email.filter(val=>val.value)
                         non_empty_values.forEach(vv=>{
-                            email_totals[group].push(window.lodash.escape(vv.value))
+                            email_totals[group].push(_.escape(vv.value))
```

Comment:
Keep it as `window.lodash.` instead of `_.`
_ conflicts with underscore with certain setups.